spec/controllers/users/backup_code_setup_controller_spec.rb

```diff
@@ -1,11 +1,33 @@
 require 'rails_helper'
 
 describe Users::BackupCodeSetupController do
-  it 'creates backup codes' do
+  it 'creates backup codes and logs expected events' do
     user = build(:user, :signed_up)
+    attributes_visited = {
+      success: true,
+      errors: {},
+      mfa_method_counts: {phone: 1},
+      pii_like_keypaths: [[:mfa_method_counts, :phone]],
+      error_details: nil,
+      enabled_mfa_methods_count: 1,
+    }
+    attributes_created = {
+      enabled_mfa_methods_count: 2,
+    }
```

Comment:
since these variables are not used... I would just inline them below in the assertions
```suggestion
```

Reply:
Understood and taken care of!